Lumberjack.xcodeproj/project.pbxproj

```diff
@@ -1410,145 +528,17 @@
-			target = 19EC14661B84D134000EC2E7 /* watchOSSwiftTest */;
-			targetProxy = 19EC147F1B84D135000EC2E7 /* PBXContainerItemProxy */;
-		};
-		C74150B2215E6D3E002D7D6F /* PBXTargetDependency */ = {
-			isa = PBXTargetDependency;
-			target = 19FF46011B8B4CF400B43179 /* CocoaLumberjack */;
-			targetProxy = C74150B1215E6D3E002D7D6F /* PBXContainerItemProxy */;
-		};
-		C74150B7215E6D50002D7D6F /* PBXTargetDependency */ = {
-			isa = PBXTargetDependency;
-			target = 19FF460E1B8B4D1400B43179 /* CocoaLumberjackSwift */;
-			targetProxy = C74150B6215E6D50002D7D6F /* PBXContainerItemProxy */;
-		};
 		C7D056751E1A5A2C000E6591 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
 			target = 19FF46011B8B4CF400B43179 /* CocoaLumberjack */;
 			targetProxy = C7D056741E1A5A2C000E6591 /* PBXContainerItemProxy */;
 		};
 /* End PBXTargetDependency section */
 
-/* Begin PBXVariantGroup section */
-		18F3BFFE1A81E0C800692297 /* Main.storyboard */ = {
-			isa = PBXVariantGroup;
-			children = (
-				18F3BFFF1A81E0C800692297 /* Base */,
-			);
-			name = Main.storyboard;
-			sourceTree = "<group>";
-		};
-		18F3C0031A81E0C800692297 /* LaunchScreen.xib */ = {
-			isa = PBXVariantGroup;
-			children = (
-				18F3C0041A81E0C800692297 /* Base */,
-			);
-			name = LaunchScreen.xib;
-			sourceTree = "<group>";
-		};
-		19406BE71BDBABFF001194DC /* Main.storyboard */ = {
-			isa = PBXVariantGroup;
-			children = (
-				19406BE81BDBABFF001194DC /* Base */,
-			);
-			name = Main.storyboard;
-			sourceTree = "<group>";
-		};
-		19EC14691B84D134000EC2E7 /* Interface.storyboard */ = {
-			isa = PBXVariantGroup;
-			children = (
-				19EC146A1B84D134000EC2E7 /* Base */,
-			);
-			name = Interface.storyboard;
-			sourceTree = "<group>";
-		};
-		DCB318D014ED6C3B001CFBEE /* InfoPlist.strings */ = {
-			isa = PBXVariantGroup;
-			children = (
-				DCB318D114ED6C3B001CFBEE /* en */,
-			);
-			name = InfoPlist.strings;
-			sourceTree = "<group>";
-		};
-		DCB318D614ED6C3B001CFBEE /* Credits.rtf */ = {
-			isa = PBXVariantGroup;
-			children = (
-				DCB318D714ED6C3B001CFBEE /* en */,
-			);
-			name = Credits.rtf;
-			sourceTree = "<group>";
-		};
-		DCB318DC14ED6C3B001CFBEE /* MainMenu.xib */ = {
-			isa = PBXVariantGroup;
-			children = (
-				DCB318DD14ED6C3B001CFBEE /* en */,
-			);
-			name = MainMenu.xib;
-			sourceTree = "<group>";
-		};
-/* End PBXVariantGroup section */
-
 /* Begin XCBuildConfiguration section */
-		18F3BF7C1A81DD2E00692297 /* Debug */ = {
-			isa = XCBuildConfiguration;
-			baseConfigurationReference = 1B1DBB6F9BB5466559BF9024 /* Pods-iOSSwiftTest.debug.xcconfig */;
-			buildSettings = {
-				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				GCC_PREPROCESSOR_DEFINITIONS = (
-					"DD_LOG_LEVEL=DDLogLevelWarning",
-					"$(inherited)",
-				);
-				INFOPLIST_FILE = "$(SRCROOT)/Framework/iOSSwift/Info.plist";
-				SDKROOT = iphoneos;
-				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos";
-				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
@lolgear see an example here of the settings that need to be explicitly set for a target. Including `DD_LOG_LEVEL` cc @ffried